src/mbgl/style/expression/within.cpp

```diff
@@ -204,6 +233,13 @@ Value valueConverter(const mapbox::geojson::rapidjson_value& v) {
         }
         return result;
     }
+    if (v.IsObject()) {
+        std::unordered_map<std::string, Value> result;
```

Comment:
nit: could we `result.reserve()` beforehand?

Reply:
Object does not contain a size info.